pkg/asset/manifests/azure/cluster.go

```diff
@@ -40,7 +42,7 @@ func GenerateClusterAssets(installConfig *installconfig.InstallConfig, clusterID
 			Namespace: capiutils.Namespace,
 		},
 		Spec: capz.AzureClusterSpec{
-			ResourceGroup: clusterID.InfraID,
+			ResourceGroup: fmt.Sprintf("%s-rg", clusterID.InfraID),
```

Comment:
this lgtm, but should be in its own commit (it is a standalone fix unrelated to the capz machine changes)